### PR TITLE
Fix #2375: Move Node and Phantom CustomInitFilesTest to jsEnvTestSuite.

### LINKED ISSUE #2375: Move CustomInitFilesTest to jsEnvTestSuite
I don't know why they ended up in the kit in the first place. At lest the Node.js and PhantomJS instantiations of the test should be in the suite not the kit.

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

**Comment by nicolasstucki** [comment]
Review by @sjrd

File: project/BinaryIncompatibilities.scala [hunk 1]
```diff
@@ -12,6 +12,8 @@ object BinaryIncompatibilities {
   )
 
   val JSEnvsTestKit = Seq(
+    ProblemFilters.exclude[MissingClassProblem]("org.scalajs.jsenv.test.NodeJSWithCustomInitFilesTest"),
+    ProblemFilters.exclude[MissingClassProblem]("org.scalajs.jsenv.test.PhantomJSWithCustomInitFilesTest")
```
Comment (sjrd) [inline project/BinaryIncompatibilities.scala]: There should be a comment saying "Breaking", as this is actually breaking.
Also, 4 spaces wrt `val`, and wrap with 4 more spaces after `(`.

**Comment by sjrd** [comment]
That's all.

**Comment by sjrd** [comment]
LGTM

**Pull request merged**